development/lint/electron.js

```
@@ -19,3 +20,25 @@ try {
   );
   exit(1);
 }
+
+// Check if APP_NAME is correctly set to "OneKey Wallet"
+  const distAppPath = path.join(desktopPath, 'app', 'dist', 'app.js');
```

ESLint

[error] Delete `··`

(prettier/prettier)